### DISCUSSION TITLE
(Discussion) About the irregular Brush tool strokes

### DISCUSSION THREAD

**Discussion by F-Burning** [opentoonz/opentoonz#6503] [Ideas]
### Description

I imagine that it has been commented many times, but the brush tool makes irregular strokes compared to the geometry ones

![brush](https://user-images.githubusercontent.com/74098734/209482154-22ed59df-b3bb-4b5e-b9a7-a7d6f23ab653.jpg)

I don't know if it's for aesthetic purposes, it's a limitation of the program, or a bug, but I think all the strokes should look the same, right?

**Comment by RodneyBaker** [comment]
Also relates to #2229
